sdk/test-utils/identity-extensions/src/index.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ClientSecretCredential } from "@azure/identity";
+import { env, isPlaybackMode } from "@azure-tools/test-recorder";
+import { NoOpCredential } from "@azure-tools/test-recorder-new";
```

Comment:
Shouldn’t NoOpCredential be moved to this package?

Reply:
Hmm, can be done